build.sbt

```diff
@@ -262,9 +263,7 @@ lazy val `interface-test` = project
     },
     Test / unmanagedClasspath ++= Def.taskDyn {
       if (sys.env.get("TEST_VERSION").isEmpty)
-        Def.task {
-          Seq((interface / finalPackageBin).value)
-	}
+        Def.task { Seq((interface / finalPackageBin).value) }
```

Comment:
Let's keep the former code here, it doesn't change anything.

Reply:
it's my mistake, I thought I was fixing bad idnentation, but turns out it was me that I had messed up the original code